### PR TITLE
Improve markdown in changes.

### PR DESCRIPTION
**Pull request by silene**
This was mostly a matter of adding backquotes and indentation where
needed. There were also some "combining grave accent" used in place of
proper backquotes.

I cleaned only the changes of the most recent releases.

### CODE REVIEW

**Comment by Zimmi48** [comment]
So @silene WDYT? Should it be time to actually add the `.md` extension so that CHANGES is rendered as Markdown?

File: CHANGES [hunk 1]
```diff
@@ -149,44 +149,43 @@ Standard Library
...
+  the command line: `COQFLAGS`, `COQCHKFLAGS`, `COQDOCFLAGS`.
+  `COQFLAGS` is now entirely separate from `COQLIBS`, so in custom Makefiles
+  `$(COQFLAGS)` should be replaced by `$(COQFLAGS) $(COQLIBS)`.
 
-- Removed the gallina utility (extracts specification from Coq vernacular files).
+- Removed the `gallina` utility (extracts specification from Coq vernacular files).
   If you would like to maintain this tool externally, please contact us.
 
 - Removed the Emacs modes distributed with Coq. You are advised to
-  use Proof-General <https://proofgeneral.github.io/> (and optionally
```
[the remark this answers is not in the document]
Comment (silene) [reply CHANGES]: Yes, that is one of the parts about "improving" the markdown. I tend to think that inline URLs make texts harder to read, so I prefer hyperlinks when they make sense.

File: dev/doc/changes.md [hunk 2]
```diff
@@ -1,12 +1,5 @@
 ## Changes between Coq 8.9 and Coq 8.10
 
-### ML API
-
-Termops:
-
-- Internal printing functions have been placed under the
-  `Termops.Internal` namespace.
-
```
[the remark this answers is not in the document]
Comment (silene) [reply dev/doc/changes.md]: Right, I forgot about it. I was using the 8.9 changelog as a canvas, to ease the backport work later.

**Comment by silene** [comment]
> Should it be time to actually add the `.md` extension so that `CHANGES` is rendered as Markdown?

As far as I am concerned, most certainly. I don't know know what was the reason not to do it earlier. Was it just force of the habit?

**Comment by Zimmi48** [comment]
>I don't know know what was the reason not to do it earlier.

My reason was to wait to have enough releases which are close enough to markdown style. I personally think it is time, yeah.

File: dev/doc/changes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zimmi48) [reply dev/doc/changes.md]: So this is on purpose, right?

**Review by Zimmi48 — COMMENTED** [review]
Please add the file renaming (maybe a separate commit on top) and I'll merge.
Note that your commit renaming the file must also update the following files:

```
.github/PULL_REQUEST_TEMPLATE.md:- [ ] Entry added in CHANGES.
README.md:There is a file named [`CHANGES`](CHANGES) that explains the differences and the
dev/build/windows/makecoq_mingw.sh:    install -D CHANGES                        "$PREFIXCOQ/license_readme/coq/Changes.txt"
dev/ci/README.md:Moreover your PR must absolutely update the [`CHANGES`](../../CHANGES) file.
dev/doc/MERGING.md:Moreover the PR must absolutely update the [`CHANGES`](../../CHANGES) file.
```

**Comment by silene** [comment]
Done.

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**